test/integration/plugins/solidfire/TestManagedSystemVMs.py

```diff
@@ -0,0 +1,586 @@
+                TestData.email: "test@test.com",
+                TestData.firstname: "John",
+                TestData.lastname: "Doe",
+                TestData.username: "test",
+                TestData.password: "test"
+            },
+            TestData.user: {
+                TestData.email: "user@test.com",
+                TestData.firstname: "Jane",
+                TestData.lastname: "Doe",
+                TestData.username: "testuser",
+                TestData.password: "password"
+            },
+            TestData.primaryStorage: {
+                TestData.name: TestData.get_name_for_solidfire_storage(),
+                TestData.scope: "ZONE",
+                TestData.url: "MVIP=192.168.139.112;SVIP=10.10.8.112;" +
+                       "clusterAdminUsername=admin;clusterAdminPassword=admin;" +
+                       "clusterDefaultMinIops=10000;clusterDefaultMaxIops=15000;" +
+                       "clusterDefaultBurstIopsPercentOfMaxIops=1.5;",
+                TestData.provider: "SolidFire",
+                TestData.tags: TestData.storageTag,
+                TestData.capacityIops: 4500000,
+                TestData.capacityBytes: 2251799813685248,
+                TestData.hypervisor: "Any",
+                TestData.zoneId: 1
+            },
+            TestData.virtualMachine: {
+                TestData.name: "TestVM",
+                "displayname": "Test VM"
+            },
+            TestData.computeOffering: {
+                TestData.name: "SF_CO_1",
+                "displaytext": "SF_CO_1 (Min IOPS = 10,000; Max IOPS = 15,000)",
+                "cpunumber": 1,
+                "cpuspeed": 100,
+                "memory": 128,
+                "storagetype": "shared",
+                "customizediops": False,
+                TestData.min_iops: 10000,
+                TestData.max_iops: 15000,
+                "hypervisorsnapshotreserve": 200,
+                TestData.tags: TestData.storageTag
+            },
+            TestData.systemOffering: {
+                TestData.name: "SF_SO_1",
+                "displaytext": "Managed SO (Min IOPS = 4,000; Max IOPS = 8,000)",
+                "cpunumber": 1,
+                "cpuspeed": 100,
+                "memory": 128,
+                "storagetype": "shared",
+                TestData.min_iops: 4000,
+                TestData.max_iops: 8000,
+                TestData.tags: TestData.storageTag,
+                "issystem": True
+            },
+            TestData.systemOfferingFailure: {
+                TestData.name: "SF_SO_2",
+                "displaytext": "Managed SO (Customized IOPS)",
+                "cpunumber": 1,
+                "cpuspeed": 100,
+                "memory": 128,
+                "storagetype": "shared",
+                "customizediops": True,
+                TestData.tags: TestData.storageTag,
+                "issystem": True
+            },
+            TestData.templateName: "CentOS 5.6(64-bit) no GUI (XenServer)",
```

Comment:
The template name should not be hardcoded.  There is a value in the `config` for the template, but I forget the exact name.  @abhinandanprateek do recall the value that should be used here?